data/templates/set_bmc_credentials.sh

```diff
@@ -47,14 +49,38 @@ for x in $cmdReturn; do
   i=$((i+1))
 done
 
+function get_newUserNumber()
+{
+        cmdReturn=$(sudo ipmitool user summary $channel)
```

Comment:
Please change the indent to the same as before. Or at least to keep the same file use the same indent.